src/github-action/index.ts

```diff
@@ -0,0 +1,17 @@
+import * as core from "@actions/core";
+
+import { VersionResolver } from "../version";
+
+import { newCliInstaller } from "./cli-installer";
```

Comment:
`[nit]`
Grouping them together makes things still clear while keeping them more compact.
```suggestion
import * as core from "@actions/core";
import { VersionResolver } from "../version";
import { newCliInstaller } from "./cli-installer";
```

Reply:
This is automatically formatted to this by running `npm run lint:fix`